src/uwtools/resources/fv3.jsonschema

```diff
@@ -11,7 +11,7 @@
     }
   },
   "properties": {
-    "forecast": {
+    "fv3": {
```

Comment:
File [config.txt](https://github.com/ufs-community/uwtools/files/14231292/config.txt) (actually contains YAML) validates against this schema and should give a good idea of the current state of the FV3 config. On `hera`, the run in `/home/Paul.Madden/run` was created with this `config.yaml` and the command `uw fv3 run --config-file config.yaml --cycle 2019-07-01T00 --batch`. That run gets as far and fails in the same way as the reference run `/scratch2/BMC/wrfruc/cholt/forecast_driver_test/expt/2019070100`, so I think the code in this PR at least reproduces the capabilities of the previous version. (Which is not to say it doesn't still need some work.)